lib/db.js

```diff
@@ -284,7 +284,7 @@ class LimitDBRedis extends EventEmitter {
             reset: Math.ceil(reset / 1000),
             limit: bucketKeyConfig.size,
             delayed: false,
-            delta_reset_ms: reset - currentMS,
+            delta_reset_ms: Math.max(reset - currentMS, 0)
```

Comment:
Tests (esp. override tests) are awful so I won't distract and block you for this, but if you decide to change anything in the PR anyway, consider writing a unit test for this at least for documentation purposes

Reply:
added